spec/truffle/identity_spec.rb

```diff
@@ -33,7 +33,8 @@
   end
 
   it "RUBY_RELEASE_DATE is the current year" do
```

Comment:
Should the description be changed as well?

Reply:
It's just an edge case.